arbeitszeit/member_notifications.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+from uuid import UUID
+
+from arbeitszeit.presenters import InviteWorkerPresenter as Presenter
+from arbeitszeit.repositories import DatabaseGateway
+
+
+@dataclass
+class MemberNotifier:
+    presenter: Presenter
+    database: DatabaseGateway
+
+    def notify_member_about_invitation(self, member: UUID) -> None:
```

Comment:
I'd argue that this method should just be a "private" method of the `InviteWorkerToCompanyUseCase`. I doubt that this method will be reused anywhere else soon.